perf/stability/multicluster-vpn/templates/locality-failover.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: srv-locality-failover
+spec:
+  ports:
+  - port: 8080
+    name: http-echo
+  - port: 8079
+    name: grpc-ping
+  selector:
+    app: srv-locality-failover
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: srv-locality-failover
+spec:
+  replicas: 4
```

Comment:
nit: maybe make configurable in helm?

Reply:
I'm planning on kustomize but this is WIP

Reply:
+1 on doing it via kustomize - I think we could just have it kustomize first, without any helm template.